aws/resource_aws_guardduty_member_test.go

```diff
@@ -33,6 +35,47 @@ func testAccAwsGuardDutyMember_basic(t *testing.T) {
 	})
 }
 
+func testAccAwsGuardDutyMember_invite(t *testing.T) {
+	resourceName := "aws_guardduty_member.test"
+	accountID := "111111111111"
+	email := "required@example.com"
+	invitationMessage := "inviting"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAwsGuardDutyMemberDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccGuardDutyMemberConfig_invite(accountID, email, invitationMessage, true, false),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAwsGuardDutyMemberExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "account_id", accountID),
+					resource.TestCheckResourceAttrSet(resourceName, "detector_id"),
+					resource.TestCheckResourceAttr(resourceName, "email", email),
+					resource.TestCheckResourceAttr(resourceName, "invite", strconv.FormatBool(true)),
+					resource.TestCheckResourceAttr(resourceName, "disable_email_notification", strconv.FormatBool(false)),
+					resource.TestCheckResourceAttr(resourceName, "invitation_message", invitationMessage),
+				),
+				ExpectError: regexp.MustCompile("Expected member to be invited but was in state: EmailVerificationFailed"),
```

Comment:
After migrating the member account ID and email to environment variables, I will fix this test on merge to not need `ExpectError` 👍